app/scripts/components/misc/t.jsx

```diff
@@ -43,6 +54,13 @@ const T = React.createClass({
     if (typeof translated === 'undefined') {
       warn('missing translation for key', this.props.k);
       translated = this.props.k;
+    } else if (typeof translated === 'function') {
+      if (!(this.propTypes.i instanceof Array)) {
+        warn('missing expected array for interpolating', this.props.k, 'got:', this.propTypes.i);
+        translated = translated();
+      } else {
+        translated = translated(...this.props.i);
```

Comment:
it's not a great way to interpolate either. It might be a good idea to switch to a library that handles this better.